arviz/plots/backends/matplotlib/lmplot.py

```diff
@@ -0,0 +1,33 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from ...hdiplot import plot_hdi
+
+def plot_lm(
+    y_model_values,
```

Comment:
Is there a way we can use inference data as an input and specify the x_values and such in reference to inf data?

Reply:
Then I guess y_model need to be stored somewhere in idata.